src/NServiceBus.AcceptanceTests/Core/CoreTestSuiteConstraints.cs

```diff
@@ -6,10 +6,10 @@ public partial class TestSuiteConstraints
     {
         public bool SupportsDtc => false;
         public bool SupportsCrossQueueTransactions => true;
-        public bool SupportsNativePubSub => true;
+        public bool SupportsNativePubSub => false;
```

Comment:
this change is for testing purposes and needs to be reverted again, but we can't test MDPS with the acceptance tests in core otherwise

Reply:
Can't we create a new test with the AcceptanceTesting transport to test this? (since it can be configured to support MDPS)

Reply:
@andreasohlund no, not really. We would have to create a dedicated test project.
technically it is possible to re-configure the transport on the test directly, but this has side-effects like missing cleanup logic and stuff like that, so I wouldn't use that.

Reply:
we do run this setting for the inmem persistence tests though. So that's probably good enough as it ensures we run mdps tests as part of the core build.